treeherder/model/migrations/0038_commit_search_vector_idx.py

```diff
@@ -13,16 +12,17 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.AddField(
+            model_name="commit",
+            name="search_vector",
+            field=django.contrib.postgres.search.SearchVectorField(
+                blank=True, null=True
+            ),
```

Comment:
should we populate it for all existing commits?

Reply:
Oh yes. I only did that locally by running
```
from django.contrib.postgres.search import SearchVector
from treeherder.model.models import Commit

Commit.objects.update(
    search_vector=SearchVector('comments', 'author', 'revision')
)

I'll have to create a migration to update existing data